laserGui.py

```diff
@@ -233,6 +258,11 @@ def setUpExistingFilter():
 Box(welcomeBox, width="fill", height=60) # spacer
 Text(welcomeBox, text="Welcome", size=72)
 Text(welcomeBox, text="Tap your fob to begin", size=36)
+syncTimeText = Text(welcomeBox, text="Last update:", size=12 )
+updateSyncTime()
+btnSyncNow = PushButton( welcomeBox, command=syncAuthList, text="Sync Now", width=10, pady=12 )
+btnSyncNow.highlightbackground = "blue"
+
```

Comment:
Ideally this would not be visible to users, it would mainly be used for troubleshooting, could we have this toggleable, configured an env variable? Default to not visible.

Reply:
yes, I agree but since we've been having trouble with the logins i wanted to expose it in case people got stuck. I wanted to put it in a less prominent place like the bottom corner but didn't want to mess with the zerogui layout too much so i left it here. 

I will put it behind a env var. so we can turn it off.